plugins/inputs/modbus/README.md

```diff
@@ -261,22 +261,21 @@ Select the type `FIXED` when the input type is declared to hold signed integer v
 of the modbus device should indicate this with a term like 'int32 containing fixed-point representation
 with N decimal places'.
 
-(FLOAT32 is deprecated and should not be used any more. UFIXED provides the same conversion
-from unsigned values).
+(FLOAT32 is deprecated and should not be used. UFIXED provides the same conversion from unsigned values).
 
 ---
 
 ### `request` configuration style
 
-This sytle can be used to specify the modbus requests directly. It allows to specify multiple `[[inputs.modbus.request]]` sections including multiple slave-devices. This way, _modbus_ gateway devices can be queried. Please not that _requests_ might be split for non-consecutive addresses. If you want to avoid this behavior please add _fields_ with the `omit` flag set filling the gaps between addresses.
+This sytle can be used to specify the modbus requests directly. It enables specifying multiple `[[inputs.modbus.request]]` sections including multiple slave-devices. This way, _modbus_ gateway devices can be queried. Please note that _requests_ might be split for non-consecutive addresses. If you want to avoid this behavior please add _fields_ with the `omit` flag set filling the gaps between addresses.
 
 #### Slave device
 
-You can use the `slave_id` setting to specify the ID of the slave device to query. It should be specified for each request and defaults to zero otherwise. Please note, only one `slave_id` can be specified for a request.
+You can use the `slave_id` setting to specify the ID of the slave device to query. It should be specified for each request. otherwise it defaults to zero. Please note, only one `slave_id` can be specified per request.
```

Comment:
```suggestion
You can use the `slave_id` setting to specify the ID of the slave device to query. It should be specified for each request. Otherwise, it defaults to zero. Please note, only one `slave_id` can be specified per request.
```